README.md

```diff
@@ -60,26 +91,39 @@ This plugin has been inspired by the work previously done by [esensar](https://g
     remove_existing_container = true,
     -- By default, if no extra config is added, following nvim_dotfiles are
     -- installed: "https://github.com/LazyVim/starter"
-    -- This is an example for configuring other nvim_dotfiles inside the docker container
-    nvim_dotfiles_repo = "https://github.com/arnaupv/dotfiles.git",
-    nvim_dotfiles_branch = "main", -- branch to clone from nvim_dotfiles_repo 
-    nvim_dotfiles_install_command = "cd ~/nvim_dotfiles/ && ./install.sh",
-    -- In case you want to change the way the devenvironment is setup, you can also provide your own setup
-    setup_environment_repo = "https://github.com/arnaupv/setup-environment",
-    setup_environment_install_command = "./install.sh -p 'nvim stow zsh'",
+    -- This is an example for configuring other dotfiles inside the docker container
+    dotfiles_repository = "https://github.com/erichlf/dotfiles.git",
+    dotfiles_branch = "main", -- branch to clone from dotfiles_repository`
+    dotfiles_targetPath = "~/dotfiles", -- location to install dotfiles
+    dotfiles_targetPath = "install.sh", -- script to run after dotfiles are cloned
   },
   keys = {
     -- stylua: ignore
     {
-      "<leader>cdu",
+      "<leader>Du",
```

Comment:
`<Leader>Du` returns a `fatal: repository 'https://github.com/devcontainer.git/' not found` in `Start: Run in container: # Clone & install dotfiles`. The container is up and running afterwards, but `dotfiles` are not installed.

Reply:
Although, this was a config error the interesting part is that "repo" doesn't exist in the code, so I am not sure where it came from. The default repo is `git@github.com:erichlf/dotfiles`. So it may be worth me looking into this. I may create a new ticket for it if I verify the same behavior.